golem-cli/tests/text.rs

```diff
@@ -143,35 +144,19 @@ fn text_component_add(
     let component_name = format!("{name} text component add");
     let env_service = deps.component_directory().join("environment-service.wasm");
     let cfg = &cli.config;
-    let component_res = cli.with_format(Format::Text).run_string(&[
+    let res = cli.with_format(Format::Text).run_string(&[
         "component",
         "add",
         &cfg.arg('c', "component-name"),
         &component_name,
         env_service.to_str().unwrap(),
     ])?;
 
-    let lines = component_res.lines().collect::<Vec<_>>();
-
-    let regex_header = Regex::new(
-        "New component created with URN ([^ ]+), version 0, and size of ([0-9]+) bytes.",
-    )
-    .unwrap();
-    assert!(regex_header.is_match(lines.first().unwrap()));
-
-    assert_eq!(
-        *lines.get(1).unwrap(),
-        format!("Component name: {component_name}.")
-    );
-    assert_eq!(*lines.get(2).unwrap(), "Exports:");
-    assert_eq!(
-        *lines.get(3).unwrap(),
-        "\tgolem:it/api.{get-environment}() -> result<list<tuple<string, string>>, string>"
-    );
-    assert_eq!(
-        *lines.get(4).unwrap(),
-        "\tgolem:it/api.{get-arguments}() -> result<list<string>, string>"
-    );
+    assert!(regex_contains(
```

Comment:
I think the chang of this file is my favorite part :) it was so hard to maintain